components/List.jsx

```diff
@@ -17,14 +17,14 @@ const List = ({ block }) => {
     {
       list_elements.map((el)=>{
         return <a href={el.href} className="tablet:grid-col-4 height-card-lg text-center
-                                            text-base-lightest font-sans-xl text-middle
-                                            padding-105 display-flex flex-justify-center
-                                            flex-align-center"
+                                            font-sans-lg text-semibold padding-105"
                   style={{ backgroundImage: `url(${el.image})`,
                            backgroundSize: "cover",  backgroundClip: "content-box",
                            textDecoration: "none" }}
                   key={el.key}>
-          <div>
+          <div className="display-flex flex-justify-start flex-align-end padding-2"
+               style={{ height: "100%", width: "100%", color: "white",
+                        backgroundImage: 'linear-gradient(rgba(0,0,0,0), rgba(0,0,0,1))'}}>
```

Comment:
I'd personally prefer extracting these inline styles into a List.module.css and [importing it](https://nextjs.org/docs/basic-features/built-in-css-support), instead of having them inline.

I'd also maybe reduce the black to .8 or .7 opacity, (I had it at .75 in the mockup I think) so that the photo is a little more clear

Reply:
(I know this is also going on super long for a relatively small thing so also feel free to not play around with the opacity haha)